android/src/ti/identity/FingerPrintHelper.java

```diff
@@ -106,32 +107,34 @@ public void stopListening()
 	public void startListening(KrollFunction callback, KrollObject obj)
 	{
 		if (!isDeviceSupported()) {
```

Comment:
The `isDeviceSupported()` method will return `false` if you're running on API Level less than 23.
Titanium's min API Level is currently 19 (aka: Android 4.4).

Can you test this on Android 4.4 please? I know that finger print is only supported on Android 6.0 and higher, but Google's docs doesn't make it clear if you can use device credentials (ie: pin, password, etc.) on the older Android OS versions.

Reply:
@jquick-axway  I test this on Android 4.4 and the fallback to device credentials does not support this android version or may be API level less than 23.